infrastructure/application/README.md

```diff
@@ -0,0 +1,18 @@
+# Application stack
+
+This stack holds our application layer, including the following services:
+- [Hasura](https://hasura.editor.planx.uk)
+- [API](https://api.editor.planx.uk)
+- [ShareDB](https://sharedb.editor.planx.uk)
+- [Editor](https://editor.planx.uk)
+- [Metabase](https://metabase.editor.planx.uk)
+
+## Custom domains
+
+In order to set up custom domains for council services:
+
+1. Refer to our [public documentation on Notion](https://www.notion.so/IT-systems-services-d4f8b88fb9694f33a24411801150c793) (see section titled "Custom Subdomains") for what information we need from councils in order to set up a custom domain for them.
+2. Add the desired subdomain to the constant `CUSTOM_DOMAINS` in [index.ts]().
+3. Look at the definition of `createCustomDomain` on [index.ts]() for instructions on how to set up SSL certificates for the new domain (i.e. using `pulumi config`).
```

Comment:
this is helpful, thank you! 

we also have some documentation related to dev tasks for setting up subdomains over here that includes the non-infrastructure code files / database fields that need to be touched too (eg `team.domain` in hasura, `PREVIEW_ONLY_DOMAINS` in editor routes) - could we link to that at the bottom of this list as a sort of "once domain is set up on pulumi, don't forget to do these steps for it to be fully active"? https://github.com/theopensystemslab/planx-team-onboarding/blob/main/README.md